src/AnimationControl.tsx

```diff
@@ -94,178 +78,170 @@ function AnimationControl({
         };
-    }, [playAnimation, onPlayAnimationChange, loopAnimation, onFitView,
-        onLoopAnimationChange, onRestart, onShowAgentIdChange, onSkipBackward,
-        onSkipForward, onStepSizeChange, showAgentId, stepSize, onTracePathsChange, tracePaths,
-        takeScreenshot, showCellId, setShowCellId, showGoals, setShowGoals, showGoalVectors,
-        setShowGoalVectors]);
+    }, [
+        playAnimation,
+        onPlayAnimationChange,
+        onSkipBackward,
+        onSkipForward,
+        onRestart,
+        stepSize,
+        onStepSizeChange,
+        loopAnimation,
+        onLoopAnimationChange,
+        onFitView,
+        showAgentId,
+        onShowAgentIdChange,
+        tracePaths,
+        onTracePathsChange,
+        takeScreenshot,
+        showCellId,
+        setShowCellId,
+        showGoals,
+        setShowGoals,
+        showGoalVectors,
+        setShowGoalVectors,
+    ]);
 
     return (
-        <Stack direction="column" spacing={1}>
-            <Stack direction="row" spacing={2} justifyContent="center">
-                <Tooltip
-                    title={
-                        <div style={{ textAlign: 'center' }}>
-                            Adjust animation step size
-                            ({STEP_SIZE_UP_KEY}/{STEP_SIZE_DOWN_KEY})
-                        </div>
-                    }
-                >
-                    <Slider
-                        value={stepSize}
-                        step={STEP_SIZE_INCREMENT}
-                        marks
-                        min={STEP_SIZE_MIN}
-                        max={STEP_SIZE_MAX}
-                        valueLabelDisplay="auto"
-                        onChange={handleSliderChange}
-                        sx={{ width: '40%', height: "auto"}}
-                    />
-                </Tooltip>
-                <Tooltip title="Reset step size">
-                    <Button onClick={() => roundAndSetStepSize(1)}>
-                        <RestartAltIcon />
-                    </Button>
-                </Tooltip>
-            </Stack>
-            <Box display="flex" justifyContent="center">
-                <ButtonGroup size="large" variant="outlined">
-                    <Tooltip title={`Backward one step (${STEP_BACKWARD_KEY})`}>
-                        <Button onClick={onSkipBackward}>
+        <Box sx={{
+            position: 'fixed',
+            bottom: 0,
+            left: 0,
+            right: 0,
+            zIndex: 1000,
+            backgroundColor: 'rgba(18, 18, 18, 0.95)',
+            backdropFilter: 'blur(20px)',
+            borderTop: '1px solid rgba(255, 255, 255, 0.1)',
+            px: 3,
+            py: 1.5,
+        }}>
+            <Stack direction="row" alignItems="center" justifyContent="space-between">
+                <Box sx={{ width: 48 }} />
+
+                <Stack direction="row" spacing={2} alignItems="center">
+                    <Tooltip title="Previous step (←)">
```

Comment:
```suggestion
                    <Tooltip title="Previous step (🡐)">
```
this character looks a bit nicer